cypress/utils/networking.js

```diff
@@ -0,0 +1,12 @@
+// this fn will wait until a gql query with queryName is executed on the aliased route
+export const waitForGQL = (alias, queryName) => {
+  const waitOnce = () => {
+    cy.wait(alias).then(xhr => {
```

Comment:
You probably want a way for this to timeout so that running cypress doesn't wait indefinitely on a network problem

Reply:
`cy.wait` defaults to a 5000ms timeout 
https://docs.cypress.io/api/commands/wait.html#Timeouts